packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -472,6 +496,7 @@ export default class SalesforceCRMService implements CRM {
 
     if (!contactsToCreate[0]?.email) {
       this.log.warn(`createContact: no attendee email found `, contactsToCreate);
+      return [];
```

Comment:
We were seeing the log `Cannot read properties of undefined (reading 'email')` in this method. If there are no contacts to create then let's continue with creating the Salesforce event